core/storage/classifier/gae_models.py

```diff
@@ -290,6 +290,9 @@ def create_multi(cls, job_exploration_mappings):
         Args:
             job_exploration_mappings: list(TrainingJobExplorationMapping). The
                 list of TrainingJobExplorationMapping Domain objects.
+
```

Comment:
(Since Github won't let me comment there)

Please remove the space between `new` and `TrainingJobExplorationMappingModel`.

Reply:
Done.